test/Scan.js

```diff
@@ -609,15 +609,34 @@ describe("Scan", () => {
 		it("Should send correct request on scan.exec", async () => {
 			scanPromiseResolver = () => ({"Items": []});
 			await Model.scan("name").eq("Charlie").attributes(["id"]).exec();
-			expect(scanParams.ProjectionExpression).to.eql("id");
+			expect(scanParams.ProjectionExpression).to.eql("#a1");
```

Comment:
Let's also add another check for `scan.ExpressionAttributeNames` here to make sure it's what we expect.